account/identity.go

```diff
@@ -132,9 +132,11 @@ func ConvertIdentityFromModel(t *Identity) *app.Identity {
 	id := t.ID.String()
 	converted := app.Identity{
 		Data: &app.IdentityData{
-			ID:       &id,
-			FullName: &t.FullName,
-			ImageURL: &t.ImageURL,
+			ID: &id,
```

Comment:
I think, you're missing the `Type` field here on the same level as the `ID` field.